event-handler-template-examples/src/main/java/no.unit.nva.events.examples/EventConducer.java

```diff
@@ -0,0 +1,40 @@
+package no.unit.nva.events.examples;
```

Comment:
This is an example of a class that accepts a custom event message( not from Lambda destinations)